core/scripts/vrfv2plus/testnet/main.go

```diff
@@ -0,0 +1,1191 @@
+		helpers.PanicErr(err)
+		tx, err := coordinator.OwnerCancelSubscription(e.Owner, uint64(*subID))
+		helpers.PanicErr(err)
+		helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
+	case "sub-balance":
+		consumerBalanceCmd := flag.NewFlagSet("sub-balance", flag.ExitOnError)
+		coordinatorAddress := consumerBalanceCmd.String("coordinator-address", "", "coordinator address")
+		subID := consumerBalanceCmd.Uint64("sub-id", 0, "subscription id")
+		helpers.ParseArgs(consumerBalanceCmd, os.Args[2:], "coordinator-address", "sub-id")
+		coordinator, err := vrf_coordinator_v2plus.NewVRFCoordinatorV2Plus(common.HexToAddress(*coordinatorAddress), e.Ec)
+		helpers.PanicErr(err)
+		resp, err := coordinator.GetSubscription(nil, *subID)
+		helpers.PanicErr(err)
+		fmt.Println("sub id", *subID, "balance:", resp.Balance)
+	case "coordinator-withdrawable-tokens":
+		withdrawableTokensCmd := flag.NewFlagSet("coordinator-withdrawable-tokens", flag.ExitOnError)
+		coordinator := withdrawableTokensCmd.String("coordinator-address", "", "coordinator address")
+		oracle := withdrawableTokensCmd.String("oracle-address", "", "oracle address")
+		start := withdrawableTokensCmd.Int("start-link", 10_000, "the starting amount of LINK to check")
+		helpers.ParseArgs(withdrawableTokensCmd, os.Args[2:], "coordinator-address", "oracle-address")
+
+		coordinatorAddress := common.HexToAddress(*coordinator)
+		oracleAddress := common.HexToAddress(*oracle)
+		abi, err := vrf_coordinator_v2plus.VRFCoordinatorV2PlusMetaData.GetAbi()
+		helpers.PanicErr(err)
+
+		isWithdrawable := func(amount *big.Int) bool {
+			data, err := abi.Pack("oracleWithdraw", oracleAddress /* this can be any address */, amount)
+			helpers.PanicErr(err)
+
+			_, err = e.Ec.CallContract(context.Background(), ethereum.CallMsg{
+				From: oracleAddress,
+				To:   &coordinatorAddress,
+				Data: data,
+			}, nil)
+			if err == nil {
+				return true
+			} else if strings.Contains(err.Error(), "execution reverted") {
+				return false
+			} else {
+				panic(err)
+			}
+		}
+
+		result := binarySearch(assets.Ether(int64(*start*2)).ToInt(), big.NewInt(0), isWithdrawable)
+
+		fmt.Printf("Withdrawable amount for oracle %s is %s\n", oracleAddress.String(), result.String())
+	case "coordinator-transfer-ownership":
+		cmd := flag.NewFlagSet("coordinator-transfer-ownership", flag.ExitOnError)
+		coordinatorAddress := cmd.String("coordinator-address", "", "v2 coordinator address")
+		newOwner := cmd.String("new-owner", "", "new owner address")
+		helpers.ParseArgs(cmd, os.Args[2:], "coordinator-address", "new-owner")
+
+		coordinator, err := vrf_coordinator_v2plus.NewVRFCoordinatorV2Plus(common.HexToAddress(*coordinatorAddress), e.Ec)
+		helpers.PanicErr(err)
+
+		tx, err := coordinator.TransferOwnership(e.Owner, common.HexToAddress(*newOwner))
+		helpers.PanicErr(err)
+
+		helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID, "transfer ownership to", *newOwner)
+	case "vrf-owner-deploy":
```

Comment:
Lets get rid of all vrf owner related cmds, they're for v2 only.

Reply:
Done. Removed the following commands:

- vrf-owner-deploy
- vrf-owner-set-authorized-senders
- vrf-owner-accept-vrf-ownership